docs_source/content/server.md

```diff
@@ -16,6 +16,32 @@ Converge comes with a server that can:
 - serve the Converge binary itself, for bootstrapping new systems inside your
   network
 
+## Basic Usage
+
+Install the converge binary to your server or device and run the `server` command.
```

Comment:
Let's have instructions here to get an actual token. The `uuid` utility produces a sufficiently random token, and is commonly available.